src/frontend/web_application/src/components/AuthorAvatar/presenter.jsx

```diff
@@ -1,5 +1,7 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { SIZE_SMALL, SIZE_MEDIUM } from '../ContactAvatarLetter';
```

Comment:
Sometimes a recipient is not a contact. May be move this into a ParticipantAvatar component ?

Reply:
If you're ok, this will be done in a next PR with moving `Avatar`s components into their own module ?